doc/observables.rst.py

```diff
@@ -44,11 +44,16 @@ def latex_to_rst(s):
         print('     - Kinematic Variables')
         for qn, entry in group:
             latex = entry.latex()
+            unit_string = entry.unit().latex()
+            if unit_string == '1':
+                unit_string = ''
+            else:
+                unit_string = r'\, \left[ {unit_string} \right]'.format(unit_string=unit_string)
```

Comment:
I think this does not do what you expect it to do. The unit's latex representation does not contain the math delimiters. You have to add them yourself. Also, ```\left[``` only works in a math environment.

Reply:
I agree on the result, but not your explanation. The magic happens in line 59, which I overlooked.
This seems fine. Did you look at the documentation it produces? ``make -C doc html`` and look into ``doc/_build/index.html`` I think.